samples/central-login/src/main.js

```diff
@@ -11,7 +11,7 @@ import * as forgerock from '@forgerock/javascript-sdk';
  */
 
 const FATAL = 'Fatal';
-
+const { AM_URL, WEB_OAUTH_CLIENT, TIMEOUT, REALM_PATH } = process.env;
```

Comment:
I see that this was added, but I don't see any of the variables used.

Reply:
yeah my bad was debugging will remove. thank you